### ISSUE TITLE
macOS cfstream test failing on master

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#18724]
This was likely introduced by https://github.com/grpc/grpc/pull/18546, see
comment https://github.com/grpc/grpc/pull/18546#issuecomment-482028019

https://source.cloud.google.com/results/invocations/74da9667-391b-48a1-9d4e-e3d6879546df

```
+ bazel test --spawn_strategy=standalone --genrule_strategy=standalone --test_output=all //test/cpp/end2end:cfstream_test
WARNING: The following rc files are no longer being read, please transfer their contents or import their path into one of the standard rc files:
/etc/bazel.bazelrc
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Invocation ID: d14c8c7c-60d4-4822-8d6f-798fee15a677
Loading:
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: Rule 'io_bazel_rules_python' modified arguments {"shallow_since": "1523916428 -0700"}
Loading: 0 packages loaded
INFO: Rule 'org_pubref_rules_protobuf' modified arguments {"commit": "1623a5a083f2a702bd62692d059353b4e06f3a5e", "shallow_since": "1532967846 +0000"} and dropped ["tag"]
Loading: 0 packages loaded
Loading: 0 packages loaded
Analyzing: target //test/cpp/end2end:cfstream_test (1 packages loaded, 0 targets configured)
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:33:1: file '@bazel_tools//tools/build_defs/cc:action_names.bzl' does not contain symbol 'ACTION_NAMES'
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:36:5: global variable 'ACTION_NAMES' is referenced before assignment.
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:49:5: global variable 'ACTION_NAMES' is referenced before assignment.
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:58:5: global variable 'ACTION_NAMES' is referenced before assignment.
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:68:5: global variable 'ACTION_NAMES' is referenced before assignment.
ERROR: /Volumes/BuildData/tmpfs/src/github/grpc/third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl:78:5: global variable 'ACTION_NAMES' is referenced before assignment.
ERROR: While resolving toolchains for target //test/cpp/end2end:cfstream_test: invalid registered toolchain '//third_party/toolchains/bazel_0.23.2_rbe_windows:cc-toolchain-x64_windows': error loading package 'third_party/toolchains/bazel_0.23.2_rbe_windows': Extension file 'third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl' has errors
ERROR: Analysis of target '//test/cpp/end2end:cfstream_test' failed; build aborted: invalid registered toolchain '//third_party/toolchains/bazel_0.23.2_rbe_windows:cc-toolchain-x64_windows': error loading package 'third_party/toolchains/bazel_0.23.2_rbe_windows': Extension file 'third_party/toolchains/bazel_0.23.2_rbe_windows/cc_toolchain_config.bzl' has errors
INFO: Elapsed time: 18.592s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (9 packages loaded, 35 targets configured)
```

**Comment by jtattermusch** [comment]
CC @rmstar

**Comment by jtattermusch** [comment]
This is the line that executes the https://github.com/grpc/grpc/blob/ea59977fbc3812774cfb3efeb4fa82db1c389ec8/tools/internal_ci/macos/grpc_run_bazel_tests.sh#L25 cfstream tests.
Perhaps we just need to bump the bazel version on macos workers?

**Comment by jtattermusch** [comment]
It seems that we just need to use bazel 0.23.2 on macos workers (I  tried building locally with bazel-0.23.2-darwin-x86_64 and the error went away).

**Comment by jtattermusch** [comment]
Should be fixed by #18726
